Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -458,3 +466,18 @@ dummyFareBreakUp = FareBreakupAPIEntity{
   amount : 0,
   description : "fare"
 }
+
+dummyBusTicket :: BusTicketRes
+dummyBusTicket = BusTicketRes{
+  id : "123456",
+  status : "CONFIRMED",
+  quoteId : Nothing,
+  paymentUrl : Nothing,
+  quantity : 2,
+  pricePerAdult : 20.0,
+  totalPrice : 40.0,
+  qrData : "nammayatri.in",
+  registrationNumber : "MH 14 YF 4921",
+  busRouteNo : "BEST BUS - 342F",
+  busModel : "Scania Higher a30"
+}
```

Comment:
data should be in such a way that in case its breaking and falling back to default condition screen should be appropriate